inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -70,75 +79,79 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
   }
 }
 
+// TODO: move this logic into a "core" bundle that comes with any theme
+// (but you only get the shadows by default in the preset theme)
+@if $bslib-dashboard-design {
 
-@mixin bslib-box-shadows {
+  :root {
+    --bslib-dashboard-sidebar-bg: var(--#{$prefix}body-bg);
+    --bslib-dashboard-sidebar-main-bg: var(--#{$prefix}body-bg);
+    --#{$prefix}card-border-color: var(--bslib-dashboard-border-color-translucent, var(--#{$prefix}border-color-translucent));
+    --bslib-sidebar-bg: var(--#{$prefix}body-bg);
+    --bslib-sidebar-fg: var(--#{$prefix}color);
+  }
 
-  // Add box-shadow & remove border from 'top-level' card()/layout_sidebar()
-  .bslib-card, .bslib-sidebar-layout {
-    // Shadows on top-level cards/sidebars, not in nested components
-    box-shadow: $box-shadow;
-    .bslib-card, .bslib-sidebar-layout {
-      box-shadow: none;
-    }
+  @if ($bslib-enable-shadows) {
```

Comment:
This doesn't need to be fixed here and now, but it's maybe worth keeping in mind for your follow-up PRs that if `$bslib-dashboard-design` is `false` and `$bslib-enable-shadows` is `true`, you don't get the box shadows. Maybe we're ok with that (I am, if there's not an easy solution), but I suppose it has the potential to be confusing for users

Reply:
Totally. I have one more commit to add to this to bring in some shadow styles from Greg and I've already moved this to make shadows and dashboards two separate toggles.